src/components/SearchNewTabSwitch/SearchNewTabSwitch.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { createStyles, Switch, Group } from '@mantine/core';
+import { useTranslation } from 'next-i18next';
+import { useConfig } from '../../tools/state';
+
+const useStyles = createStyles((theme) => ({
+  root: {
+    position: 'relative',
+    '& *': {
+      cursor: 'pointer',
+    },
+  },
+
+  icon: {
+    pointerEvents: 'none',
+    position: 'absolute',
+    zIndex: 1,
+    top: 3,
+  },
+
+  iconLight: {
+    left: 4,
+    color: theme.white,
+  },
+
+  iconDark: {
+    right: 4,
+    color: theme.colors.gray[6],
+  },
+}));
+
+export function SearchNewTabSwitch() {
+  const { config, setConfig } = useConfig();
+  const { classes, cx } = useStyles();
+  const defaultPosition = config?.settings?.searchNewTab || '_self';
+  const [widgetPosition, setWidgetPosition] = useState(defaultPosition);
+  const { t } = useTranslation('settings/general/search-new-tab');
+  const toggleWidgetPosition = () => {
+    const position = widgetPosition === '_self' ? '_blank' : '_self';
+    setWidgetPosition(position);
+    setConfig({
+      ...config,
+      settings: {
+        ...config.settings,
+        searchNewTab: position,
+      },
+    });
+  };
+
+  return (
+    <Group>
+      <div className={classes.root}>
+        <Switch
+          checked={widgetPosition === '_blank'}
```

Comment:
If you'd use a boolean instead of a string, you wouldn't have to use the equality operator and compare to ``_blank``. See my other comments on the Settings interface.

Reply:
I changed it to boolean as suggested so this is unnecessary.